lib/ProductOpener/Display.pm

```diff
@@ -3898,6 +3898,11 @@ HTML
 						$user_template_data_ref->{edit_profile} = 1;
 						$user_template_data_ref->{orgid} = $orgid;
 					}
+					my @org_members;
```

Comment:
This feature should be available only to moderator of the pro platform, so we should have a condition on the value of $User{pro_moderator}, if the user is not a moderator, we don't create the structure with org members details.